include/deal.II/matrix_free/cuda_matrix_free.h

```diff
@@ -93,12 +93,17 @@ namespace CUDAWrappers
 
     struct AdditionalData
     {
+      /**
+       * Standardized data struct to pipe additional data to MatrixFree.
```

Comment:
Did you mean to document the `struct` instead of the constructor?

Reply:
Yes, thanks